src/adapter/src/coord/sequencer/inner/peek.rs

```diff
@@ -687,7 +721,7 @@ impl Coordinator {
 
                 let stage = match pipeline() {
                     Ok((global_lir_plan, used_indexes)) => {
-                        if let Some(explain_ctx) = explain_ctx {
+                        if let PeekContext::Explain(explain_ctx) = peek_ctx {
```

Comment:
I would have expected to see the CopyTo variant somewhere here?

Reply:
Updated!